pkg/runtime/runtime.go

```diff
@@ -588,6 +598,28 @@ func (a *DaprRuntime) getSubscribedTopicsFromApp() []string {
 	}
 	return topics
 }
+func (a *DaprRuntime) initExporters() error {
+	for _, c := range a.components {
+		if strings.Index(c.Spec.Type, "exporter") == 0 {
```

Comment:
nitpick: I know we have used this pattern, I would use the below style to reduce the long indenting... how do you think? 

```
for _, c := range a.components {
   if strings.Index(c.Spec.Type, "exporter") != 0 {
      continue
   }

   exporter, err := a.exporterRegistry.CreateExporter(c.Spec.Type)
   if err != nil {
       log.Warnf("error creating exporter %s: %s", c.Spec.Type, err)
      continue
  }
  ...
}
```

Reply:
we can sweep the code later if we want to converge on the style